examples/animation/src/lib.rs

```diff
@@ -0,0 +1,172 @@
+use std::{path::Path, sync::Arc};
+
+const SAMPLE_COUNT: rend3::types::SampleCount = rend3::types::SampleCount::One;
+
+struct AnimationExample {
+    loaded_scene: Option<rend3_gltf::LoadedGltfScene>,
+    loaded_instance: Option<rend3_gltf::GltfSceneInstance>,
+    animation_data: Option<rend3_anim::AnimationData>,
+    directional_light_handle: Option<rend3::types::DirectionalLightHandle>,
+    animation_time: f32,
+    last_frame_time: instant::Instant,
```

Comment:
You can put all of these in a single Option<AnimationInnerData> or something, and save a bunch of annoying unwraps (and you can derive default)

Reply:
I think that looks a bit better :+1: